ros2topic/ros2topic/verb/list.py

```diff
@@ -22,7 +23,7 @@ class ListVerb(VerbExtension):
     """Output a list of available topics."""
 
     def add_arguments(self, parser, cli_name):
-        add_arguments(parser)
+        add_strategy_node_arguments(parser)
```

Comment:
This change doesn't seem to be related to the topic of the PR. Please consider separating it.

Reply:
I think this comment still applies.

Reply:
Argh, I thought I had changed them all. See https://github.com/ros2/ros2cli/pull/494/commits/ba6248b11201436e21e9cccee83ac291e29698c2.